pyirf/perf/irf_maker.py

```diff
@@ -523,6 +561,58 @@ def _make_hdu(cls, hdu_name, t, cols):
         hdu.header.set("EXTNAME", hdu_name)
         return hdu
 
+    @classmethod
+    def _make_aeff_hdu(cls, table_energy, table_theta, aeff):
+        """List of columns"""
```

Comment:
See https://github.com/open-gamma-ray-astro/gamma-astro-data-formats/blob/master/source/irfs/full_enclosure/aeff/create_example_file.py

for a more concise way of creating these BinaryHDUs if you already have astropy tables.

Reply:
It looks cleaner indeed.
If you can address that comment @JouvinLea that'd be great.